### DISCUSSION TITLE
[2022/01/26] KubeSphere community meeting

### DISCUSSION THREAD

**Discussion by zryfish** [kubesphere/kubesphere#4647] [Community Meeting]
Topic: KubeSphere v3.3 Project
Location: Online
Time: 2022/01/26 16:00 PM Beijing Time
Tencent Meeting: 643-8877-6376

会议主题：KubeSphere Community Meeting
会议时间：2022/01/26 16:00-17:00 (GMT+08:00) 中国标准时间 - 北京
点击链接入会，或添加至会议列表：
https://meeting.tencent.com/dm/HOlnPLhwtQLu
#腾讯会议：643-8877-6376

**Comment by JohnNiang** [comment]
Related topic link: https://github.com/orgs/kubesphere/projects/6

**Comment by benjaminhuo** [comment]
[Fluentd Operator PR](https://github.com/fluent/fluentbit-operator/pull/189) intro & demo, Talon

**Comment by FeynmanZhou** [comment]
The meeting ID should be 643-8877-6376
